bundles/core-input-events/input-events/confirm-new-password.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+/**
+ * Account password confirmation station
+ */
+module.exports = (srcPath) => {
+  const EventUtil = require(srcPath + 'EventUtil');
+
+  return {
+    event: state => (socket, args) => {
+      const write = EventUtil.genWrite(socket);
+
+      if (!args.dontwelcome) {
+        write("<cyan>Confirm your password:</cyan> ");
+        socket.toggleEcho();
+      }
+
+      socket.once('data', pass => {
+        socket.toggleEcho();
+
+        if (!args.account.checkPassword(pass.toString().trim())) {
+          write("<red>Passwords do not match.</red>\r\n");
+          return socket.emit('new-account-password', socket, args);
+        }
+
+        return socket.emit('choose-character', socket, { account: args.account });
```

Comment:
So it looks like you updated `new-account-password` to go to `confirm-new-password` instead of `create-player` but `confirm-new-password` goes to `choose-character` instead of `create-player`. I haven't run this but just reading the code it would seem you can't create new characters